### DISCUSSION TITLE
These are my attempts so far...

### DISCUSSION THREAD

**Discussion by jaynitor** [Sakati84/3DPrintColorizer#11] [Show and tell]
Thank you for all the work you have done! Spotted this project on reddit and it has certainly got me thinking about what is possible with a modification like this. 
![IMG20210724011730](https://user-images.githubusercontent.com/5343763/126805117-cb1c04e1-e43f-4782-8c48-745328b6cd96.jpg)
Using some unbranded raw PLA+ i had left over, its quite silky and i have a feeling the pens effect it a little more then what normal pla would (possibly what ever it is mixed with i guess) but it seems to do the job.

On the Left is my first attempt i left going last night, As you can see, something went very wrong, but it showed me how it worked and i made a few tweaks to my settings so that all the pens would touch among other things.
On the Right is my last attempt, i set it so that both bottom and top layers printed on the first extruder and only the outer wall printed in colour by setting internal walls to be printed by the first extruder. I am quite happy with the result but keen to see what some playing with settings will produce.

I am currently printing a Cheshire Cat (https://www.thingiverse.com/thing:3035410) with only the internal walls coloured. I am trying to create a pocket of colour between the external wall and the infall walls. I am hoping to create a clear glaze look and trap the marker inside to prevent rubbing off when handled. Have no idea if it will work but i will hopefully report back soon. 

Thanks again Sakati84!

**Comment by Sakati84** [comment]
Awesome! Thanks for sharing your results. Did you use my Cura profile or did you create your own? If yes - what did you change?
I was thinking on only coloring the inner walls too. Unfortunately I don‘t see a way to do something similar for the top & bottom layers.

**Reply by jaynitor** [reply]
I did use your Cura profile, but found that some of the setting didn't transfer over correctly for what ever reason. I am not familiar with Cura, i have been using a fork of PrusaSlicer (SuperSlic3r) for a while now so it could have just been me not knowing what i was doing. I will report back to you when i'm getting consistent results and have some more filament to play with. Had some power outages so that ruined my last few attempts, after a resume one was even able to knock my build plate to the floor while i was not watching haha, so i will have to re level the bed and check the end stops just to make sure. Good thing the tempered glass is pretty tough heh, had some pretty drawings all over the build plate. Some of the pens did get warn a bit from drawing on the plate, I have just adjusted the z offset to counter that hoping that the differently warn nibs wont effect the end result too much.

As for the failed attempts, i am not so sure about having the outer layer un coloured, seems to wash out the colours having that layer in front. Might work with a clear glow in the dark filament to get the effect, i do have some spare i will try in the coming week when the power becomes stable as it is a little bit too pricey for me to waste on a power failure :P I was going to play around with the top surface setting to see if i can get a clear layer over the top as an extruder can be specified...Tho i could not find an option for the bottom layer unfortunately.

thanks again!

**Reply by Sakati84** [reply]
I will recheck if my profile exported correctly. I don't have much experience with Cura setting handling either.

**Reply by csanii123** [reply]
How long did it take to print the bird multicolor ? Cura estimated 4 hrs but that didn't factor in the pen changes and coloring. I did a very basic and much smaller model using a cube, cylinder and tube that Cura said 1hr but after 2hrs of printing it was maybe half done. Either I have a setting wrong or based on my machine the bird would be 16+ hrs.

**Reply by Sakati84** [reply]
The bird took 8hrs instead of 4hrs without colors for me. Did you increase your Z-axis speed as described in the Wiki?

**Reply by csanii123** [reply]
I hadn't increased the Z-axis on purpose as I wanted to see default speed before doing a faster speed. I also wanted to read up more on what this would do to my printer as I didn't want to cause issues with it. That being said the model I made was only 49 layers vs the bird which is 266 so I expected at least a 5x difference with single color so multicolor at best would be over 2x that.

**Reply by Sakati84** [reply]
I understand your reasoning but without increasing the Z speed it's way to slow.

**Reply by csanii123** [reply]
Most of my prints are pretty small so my idea is to put an adjustable lower crossbeam that would have the pen rack attached. I hope to cut the travel distance down significantly so less time for pen changes and also less wear on the z-axis motor. Will have to see if this is actually stable enough to be viable.
